### PR TITLE
Fix copy/paste

### LINKED ISSUE #3: Command Copy Selected and Paste doesn't work
Tried "Copy selected and paste ...", doesn't work. 

I think the copy one does have a bug?
It either says "command execCopy not found" or "command paste-replaced:pasteSelectionReplaced not found"

_Originally posted by @KcZer0 in https://github.com/PEZ/paste-replaced/issues/2#issuecomment-1312051966_

### LINKED ISSUE #4: Nothing is working?
After setting up settings with:
```json
"paste-replaced.replacers": [
        {
            "name": "Quote strings and newlines",
            "replacements": [
                [ "a", "b", "g" ],
            ]
        }
    ],
```

When trying default `ctrl+alt+v space` VSCode throws notification with `command 'execPaste' not found` error.
When trying default `Paste Replaced: Copy Selected Text and Paste Replaced...` VSCode throws `command 'execCopy' not found` in an error window.
After that I've also tried adding custom keybindings (and removing default ones) in keybindings.json:
```json
    {
        // paste text from clipboard into terminal and remove newline at the end if exists
        "key": "ctrl+alt+v",
        "command": "paste-replaced.paste",
        // "when": "terminalFocus",
        "args": {
            "replacements": [
                [ "\n$", "", "g" ]
            ]
        }
    },
    {
        "key": "ctrl+alt+v space",
        "command": "-paste-replaced.paste"
    },
    {
        "key": "ctrl+alt+v escape",
        "command": "-paste-replaced.interruptTyping",
        "when": "paste-replaced.isTyping"
    },
```
And that throws same error as 1st one - `command 'execPaste' not found` both in terminal and normal editors.

I've tried both on normal local VSCode (Windows 21H2) and via official Remote Host Extension (client-Windows 21H2, host-Ubuntu 20.04) - same situation everywhere.

IDK if this is supposed to be like that but Paste Replaced in VSCode's output prints only `Paste Replace activating`

VSCode version - 1.73.1
Paste Replaced version - v1.1.8

### PR DESCRIPTION
**Pull request by KcZer0**
Fixes #3, Fixes #4

### CODE REVIEW

**Comment by KcZer0** [comment]
Should be the right fix since it's the command used in default keybinding for copy/paste. 
Please do test them first before merging.

File: src/paste_replaced/replacer.cljs [hunk 1]
```diff
@@ -66,7 +66,7 @@
           (fn [c]
             (when-not (:typing-interrupted? @db/!app-db)
               (p/do! (vscode/env.clipboard.writeText c)
-                     (vscode/commands.executeCommand "execPaste")
+                     (vscode/commands.executeCommand "editor.action.clipboardCopyAction")
```
Comment (PEZ) [inline src/paste_replaced/replacer.cljs]: We need to paste here. 😄 
```suggestion
                     (vscode/commands.executeCommand "editor.action.clipboardPasteAction")
```

File: src/paste_replaced/replacer.cljs [hunk 2]
```diff
@@ -119,7 +119,7 @@
     (if (= simulate-typing-config "instant")
       (p/do! (vscode/env.clipboard.writeText new-text)
              (when-not skip-paste?
-               (vscode/commands.executeCommand "execPaste")))
+               (vscode/commands.executeCommand "editor.action.clipboardCopyAction")))
```
Comment (PEZ) [inline src/paste_replaced/replacer.cljs]: Here too.

```suggestion
               (vscode/commands.executeCommand "editor.action.clipboardPasteAction")))
```

**Review by PEZ — APPROVED** [review]
Thanks so much for investigating this and fixing it! It works great on my Mac as well. And also in the prompts and such.

File: src/paste_replaced/replacer.cljs (hunk 2 above)
Comment (KcZer0) [reply src/paste_replaced/replacer.cljs]: Whoops, forgot to change them 😂, thanks

**Pull request merged**